sqlmesh/dbt/loader.py

```diff
@@ -80,23 +83,27 @@ def _load_models(
         cache = DbtLoader._Cache(self, project, macros_max_mtime, yaml_max_mtimes)
 
         # First render all the config and discover dependencies
+        logger.info("Rendering model configs")
         for package in project.packages.values():
             context.variables = package.variables
 
             package.sources = {k: v.render_config(context) for k, v in package.sources.items()}
             package.seeds = {
-                k: cache.get_or_load_seed_config(v.path, lambda: v.render_config(context))
+                k: cache.get_or_load_seed_config(v.path, lambda: self._render_config(v, context))
                 for k, v in package.seeds.items()
             }
             package.models = {
-                k: cache.get_or_load_model_config(v.path, lambda: v.render_config(context))
+                k: cache.get_or_load_model_config(v.path, lambda: self._render_config(v, context))
                 for k, v in package.models.items()
             }
 
             context.add_sources(package.sources)
             context.add_seeds(package.seeds)
             context.add_models(package.models)
 
+        logger.info("Done rendering model configs")
```

Comment:
I think 2 log messages with the same log level in a row is redundant.  Observing the second message automatically means that we were done with rendering